string frame_id
string topic_suffix
float64 radius
std_msgs/ColorRGBA color
geometry_msgs/Point[] positions
--- 
geometry_msgs/Point clicked_position
---
